Sources/FileSystem/Model/BetaGroup.swift

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import Foundation
+
+public struct BetaGroup: Codable, Equatable {
+
+    public typealias EmailAddress = String
+
+    public let id: String?
+    public let groupName: String
+    public let isInternal: Bool?
+    public let publicLink: String?
+    public let publicLinkEnabled: Bool?
+    public let publicLinkLimit: Int?
+    public let publicLinkLimitEnabled: Bool?
+    public let creationDate: String?
+    public let testers: [EmailAddress]
+
+    public init(
+        id: String?,
+        groupName: String,
+        isInternal: Bool?,
+        publicLink: String?,
+        publicLinkEnabled: Bool?,
+        publicLinkLimit: Int?,
+        publicLinkLimitEnabled: Bool?,
+        creationDate: String?,
+        testers: [String] = []
+    ) {
+        self.id = id
+        self.groupName = groupName
+        self.isInternal = isInternal
+        self.publicLink = publicLink
+        self.publicLinkEnabled = publicLinkEnabled
+        self.publicLinkLimit = publicLinkLimit
+        self.publicLinkLimitEnabled = publicLinkLimitEnabled
+        self.creationDate = creationDate
+        self.testers = testers
+    }
+}
+
+extension BetaGroup: Hashable {
+    public static func == (lhs: BetaGroup, rhs: BetaGroup) -> Bool {
+        return lhs.id == rhs.id &&
+            lhs.groupName == rhs.groupName &&
```

Comment:
Why are we not including all the properties of the struct?

Reply:
Because only these properties are editable. When these properties are matching, we can assume that `group1 == group2`
Some other properties are fixed and can't be changed, so if the user changes some fixed properties like public link URL, the comparison logic will still treat them as the same thing